book/partner-resources/prep-for-kickoff.md

```diff
@@ -0,0 +1,35 @@
+# Getting Started with Building a Public Plugin
+### Partner Documentation
+
+### Purpose
+Public plugins are presented to all customers in the marketplace and are openly available for use. This document outlines the initial steps of preparing for developing a public plugin and preparing for the development kickoff call with marketplace that will help get things started.
+
+### Checklist
+
+- If you haven’t established a relationship with our Business Development team yet, reach out to Erik Koland (e.kolad@applicaster.com) in order to do so. This can help establish a mutually beneficial relationship so you can best capitalize on the opportunities our marketplace enables.
```

Comment:
I think that we need to create a special email for the Business Development team.
It needs to be generic. We can rely on one person as talented as he's. :)

Reply:
Yes, I've asked Erik for this. Originally, he was the only BizDev person. I'll check in here.